modules/core/include/opencv2/core/cvdef.h

```diff
@@ -307,10 +307,16 @@ enum CpuFeatures {
 #define CV_2PI 6.283185307179586476925286766559
 #define CV_LOG2 0.69314718055994530941723212145818
 
+#if defined (__ARM_FP16_FORMAT_IEEE) || defined (__ARM_FP16_FORMAT_ALTERNATIVE)
```

Comment:
ARM alternative format is a special format, so if we are going to use FP16 in OpenCV, I think it's better not to support the ```__ARM_FP16_FORMAT_ALTERNATIVE``` and only general IEEE format